bin/sdk/gutenberg.js

```diff
@@ -72,7 +72,9 @@ exports.compile = args => {
 	};
 
 	const name = path.basename( path.dirname( options.editorScript ).replace( /\/$/, '' ) );
-	const baseConfig = getBaseConfig( { extensionName: name } );
+	const baseConfig = getBaseConfig( {
+		extensionName: options.namespace ? name : false
```

Comment:
With `--namespace`, this would result in:

```
.tiled-gallery .wp-block-a8c-tiled-gallery .tiled-gallery-square
```

and 

```
.jetpack .wp-block-a8c-tiled-gallery .tiled-gallery-square
```

is that expected?

If so, wouldn't it be more robust if we have the `namespace` option as a string instead, which would allow us to specifically set the namespace as we use the SDK from the CLI?

Reply:
Seems like @dmsnell is opting for removing it in #26680.